src/math/Vector2.js

```diff
@@ -365,6 +367,20 @@ class Vector2 {
 
 	}
 
+	angleTo( v ) {
+
+		const denominator = Math.sqrt( this.lengthSq() * v.lengthSq() );
+
+		if ( denominator === 0 ) return Math.PI / 2;
+
+		const theta = this.dot( v ) / denominator;
+
+		// clamp, to handle numerical problems
+
+		return Math.acos( MathUtils.clamp( theta, - 1, 1 ) );
```

Comment:
You copied the code over from `Vector3` however please check if the 2D case can be implemented with less instructions: https://stackoverflow.com/questions/14066933/direct-way-of-computing-clockwise-angle-between-2-vectors